dZoneManager/Zone.cpp

```diff
@@ -403,10 +389,17 @@ void Zone::LoadPath(std::istream& file) {
 			}
 		}
 	} else if (path.pathType == PathType::Property) {
-		int32_t unknown;
-		BinaryIO::BinaryRead(file, unknown);
+
+		int32_t propertyPathType;
+		BinaryIO::BinaryRead(file, propertyPathType);
+		path.property.pathType = PropertyPathType(propertyPathType);
```

Comment:
I believe we could shorten this to BinaryIO::BinaryRead(file, path.property.pathType);
Does this work here?

Reply:
ahhh you're correct, was under the impression that due to the enum class that you have to declare it like this